button/button.css

```diff
@@ -0,0 +1,57 @@
+.btn-primary {
+	color: var(--color-white);
+	background-color: var(--color-blue-800);

```

Comment:
Add name colors as more readable. color-blue-800 looks less human readable. If its a dark shade you can mention it as dark blue or some think that

Reply:
Changes done